### PR TITLE
task: refresh page

### PR DESCRIPTION
**Pull request by adrienthebo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): task 2.6.2**

### CODE REVIEW

File: pages/common/task.md [hunk 1]
```diff
@@ -1,24 +1,36 @@
 # task
 
-> TODO list manager.
-> More information: <https://manned.org/task>.
+> Manage your TODO list from the command line.
```
Comment (CleanMachine1) [inline pages/common/task.md:3]: I prefer the old description.

File: pages/common/task.md [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+> More information: <https://taskwarrior.org/docs/>.
 
-- Add new task:
+- Add a new task:
```
Comment (CleanMachine1) [inline pages/common/task.md:6]:
```suggestion
- Add a new task which is due tomorrow:
```

File: pages/common/task.md [hunk 3]
```diff
@@ -1,24 +1,36 @@
...
 
-`task add {{thing_to_do}}`
+`task add {{description}} due:{{tomorrow}}`
 
-- List tasks:
+- Modify a task:
```
Comment (CleanMachine1) [inline pages/common/task.md:10]:
```suggestion
- Modify a task to a specific priority:
```

File: pages/common/task.md [hunk 4]
```diff
@@ -1,24 +1,36 @@
...
 
-`task list`
+`task {{task_id}} modify priority:{{H|M|L}}`
 
-- Mark task as completed:
+- Mark a task as completed:
```
Comment (CleanMachine1) [inline pages/common/task.md:14]:
```suggestion
- Mark a task as done:
```

**Comment by CleanMachine1** [comment]
Remember to check https://github.com/tldr-pages/tldr/issues/8321 out :)

File: pages/common/task.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/task.md:3]: I somehow agree. "Manage **your** TODO list" sounds a little like it is a client for some service like `todoist`. We could change it to this tough:
```suggestion
> Command-line TODO list manager.
```

File: pages/common/task.md (hunk 3 above)
Comment (marchersimon) [reply pages/common/task.md:10]: Or how about
```suggestion
- Update a task's priority:
```

Comment (kbdharun) [reply pages/common/task.md:10]: Yeah, the latter seems good.

File: pages/common/task.md (hunk 1 above)
Comment (kbdharun) [reply pages/common/task.md:3]: Do we need to have TODO in caps?
Can't it be 
```suggestion
> Command-line to-do list manager.
```

Comment (marchersimon) [reply pages/common/task.md:3]: Sure, `to-do` works too. We just have to be consistent.

File: pages/common/task.md [hunk 5]
```diff
@@ -1,24 +1,36 @@
...
-`task {{task_id}} modify {{new_thing_to_do}}`
+`task {{task_id}} delete`
 
-- Delete task:
+- List all open tasks:
 
-`task {{task_id}} delete`
+`task list`
+
+- List open tasks due before the end of the week:
```
[the remark this answers is not in the document]
Comment (CleanMachine1) [reply pages/common/task.md:26]: This definitely reads better with the before. I think we should keep it.

[the remark this answers is not in the document]
Comment (adrienthebo) [reply pages/common/task.md:26]: I agree, "open tasks due the end of week" feels fragmented.

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/task.md:26]: > I agree, "open tasks due the end of week" feels fragmented.

+1, I too think using `before` would be good.

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**